src/FormattingToolbar/toolbarMethods.js

```diff
@@ -194,9 +194,11 @@ export const transformListToBlockQuote = (editor, type, value) => {
  */
 /* eslint no-unused-expressions: 0 */
 export const transformPtoBQSwap = (editor, type) => {
-  isSelectionInput(editor.value, CONST.BLOCK_QUOTE)
+  console.log('hi')
```

Comment:
Get rid of this little guy.

Reply:
oopsie

Reply:
The little guy is gone, Please check it now!